bidding-impl/src/test/java/com/example/auction/bidding/impl/AuctionEntityTest.java

```diff
@@ -485,6 +486,61 @@ public void testInitializeCancelled() {
         assertThat(outcome2.getReplies(), hasItem(reply(PlaceBidStatus.CANCELLED, 2000, bidder1)));
     }
 
+
+    @Test
+    public void testIncrementBidOverReserveFromBellowReserve() {
+	driver.run(new StartAuction(auction), new PlaceBid(500, bidder1));
+	
+	Outcome<AuctionEvent, AuctionState> outcome = driver.run(new PlaceBid(4000, bidder1));
+
+        assertThat(outcome.events(), allOf(
+                hasItem(bidPlaced(bidder1, 2000, 4000)),
+                hasSize(1)
+        ));
+        assertThat(outcome.state().getBiddingHistory(), allOf(
+                hasItem(bid(bidder1, 2000, 4000)),
+                hasSize(1)
+        ));
+        assertThat(outcome.getReplies(), hasItem(reply(PlaceBidStatus.ACCEPTED, 2000, bidder1)));
+    }
+
+
+    @Test
+    public void testIncrementBidBellowReserveFromBellowReserve() {
+        driver.run(new StartAuction(auction), new PlaceBid(500, bidder1));
+	
+	Outcome<AuctionEvent, AuctionState> outcome = driver.run(new PlaceBid(900, bidder1));
+
+        assertThat(outcome.events(), allOf(
+                hasItem(bidPlaced(bidder1, 900, 900)),
+                hasSize(1)
+        ));
+        assertThat(outcome.state().getBiddingHistory(), allOf(
+                hasItem(bid(bidder1, 900, 900)),
+                hasSize(1)
+        ));
+        assertThat(outcome.getReplies(), hasItem(reply(PlaceBidStatus.ACCEPTED, 900, bidder1)));
```

Comment:
I think the status here should be `ACCEPTED_BELOW_RESERVE`, since the bid has been accepted, the price has been increased, but it's still below the reserve.